packages/docusaurus-plugin-content-docs/src/docs.ts

```diff
@@ -177,6 +177,8 @@ async function doProcessDocMetadata({
   // pagination buttons... frontMatter.title should be used in priority over
   // contentTitle (because it can contain markdown/JSX syntax)
   const title: string = frontMatter.title ?? contentTitle ?? baseID;
+  // To be passed to `<PageMetadata>`. Will override the title.
+  const title_meta: string = frontMatter.title_meta ?? frontMatter.title ?? contentTitle ?? baseID;
```

Comment:
We don't need any of that for the docs plugin


`title: SEO Tilte`, `sidebar_label: '...'`, `pagination_label: '...'` and `# Rendered **doc** title` are already enough, unless proven otherwise